Specify Agent Name

nhmathis opened the pull request:
- Update to specify agent name
- Persist agent on multiple docker-compose ups
- Critical for saving new demo images and maintain UCD configuration

Commit:
Specify Agent Name

- Update to specify agent name
- Persist agent on multiple docker-compose up
- Critical for saving new images and maintain UCD configuration for
demos

adamkingit commented on compose/Deploy/docker-compose.yaml:
```diff
@@ -20,6 +20,8 @@ services:
       - 7918:7918
```
We should probably remove the ^Ms from this file. Sorry we let them leak into it in the first place.

nhmathis replied:
Can do

nhmathis replied:
https://help.github.com/articles/dealing-with-line-endings/ 

GitHub's solution

Commit:
Update Line Endings

Commit:
Maintain original functionality by default

- Renamed NAME > AGENT_NAME
- If AGENT_NAME is not specified, it will use original functionality

nhmathis commented:
-^M have been removed
-If `AGENT_NAME` is not specified, the original functionality will be used. Specifying AGENT_NAME will allow reuse of agents for UCD configurations.
